hq/app/schedule/unrecognised/IamUnrecognisedUserJob.scala

```diff
@@ -77,11 +77,15 @@ class IamUnrecognisedUserJob(
 
   private def sendNotification(accountCrd: (Account, Seq[VulnerableUser]), testMode: Boolean, topicArn: String): Attempt[Option[String]] = {
     val (account, users) = accountCrd
-    if (users.isEmpty) {
+    val usersWithAtLeastOneEnabledKeyOrHuman = users.filter( user =>
+      user.key1.keyStatus == AccessKeyEnabled ||
+        user.key2.keyStatus == AccessKeyEnabled ||
+        user.humanUser
```

Comment:
As we worked out (eventually) yesterday, this condition will always be true because the vulnerable users passed to this function are always human _however_ because this is pretty fragile maybe we are better keeping this in, in case something changes in the future. Hopefully it'll be refactored away before a bug could be introduced, but I don't think it hurts?

Reply:
Yes good point @jfsoul it's worth noting that we are anticipating doing a significant refactor of this job along with the old credentials job, so we will get an opportunity to write this much more elegantly soon.